cmd/podmanV2/images/inspect.go

```diff
@@ -1,71 +1,44 @@
 package images
 
 import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
 	"strings"
+	"text/tabwriter"
+	"text/template"
 
 	"github.com/containers/buildah/pkg/formats"
+	"github.com/containers/libpod/cmd/podmanV2/common"
 	"github.com/containers/libpod/cmd/podmanV2/registry"
 	"github.com/containers/libpod/pkg/domain/entities"
-	"github.com/containers/libpod/pkg/util"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 )
 
 var (
-	inspectOpts = entities.ImageInspectOptions{}
-
 	// Command: podman image _inspect_
 	inspectCmd = &cobra.Command{
 		Use:     "inspect [flags] IMAGE",
 		Short:   "Display the configuration of an image",
 		Long:    `Displays the low-level information on an image identified by name or ID.`,
-		PreRunE: populateEngines,
-		RunE:    imageInspect,
+		RunE:    inspect,
 		Example: `podman image inspect alpine`,
 	}
-
-	containerEngine entities.ContainerEngine
+	inspectOpts *entities.InspectOptions
 )
 
-// Inspect is unique in that it needs both an ImageEngine and a ContainerEngine
-func populateEngines(cmd *cobra.Command, args []string) (err error) {
-	// Populate registry.ImageEngine
-	err = preRunE(cmd, args)
-	if err != nil {
-		return
-	}
-
-	// Populate registry.ContainerEngine
-	containerEngine, err = registry.NewContainerEngine(cmd, args)
-	return
-}
-
 func init() {
 	registry.Commands = append(registry.Commands, registry.CliCommand{
 		Mode:    []entities.EngineMode{entities.ABIMode, entities.TunnelMode},
 		Command: inspectCmd,
 		Parent:  imageCmd,
 	})
-
-	flags := inspectCmd.Flags()
-	flags.BoolVarP(&inspectOpts.Latest, "latest", "l", false, "Act on the latest container podman is aware of")
-	flags.BoolVarP(&inspectOpts.Size, "size", "s", false, "Display total file size")
-	flags.StringVarP(&inspectOpts.Format, "format", "f", "", "Change the output format to a Go template")
-
-	if registry.EngineOptions.EngineMode == entities.ABIMode {
-		// TODO: This is the same as V1.  We could skip creating the flag altogether in V2...
-		_ = flags.MarkHidden("latest")
-	}
+	inspectOpts = common.AddInspectFlagSet(inspectCmd)
```

Comment:
The latest flag has no meaning for `podman image inspect` so we shouldn't be adding it here